src/main/java/pl/edu/wat/pze/kwejk/controller/UploadController.java

```diff
@@ -40,15 +42,18 @@ public String add(@RequestParam("file") MultipartFile file,
                       BindingResult bindingResult,
                       Model model) {
         model.addAttribute(ModelAttributeEnum.ACTIVE_VIEW.toString(), ViewEnum.UPLOAD);
+
         if (bindingResult.hasErrors())
             return "index";
+
         PicValidEnum validateResult = pictureValidator.validateImageFile(file);
         if (validateResult == PicValidEnum.OK) {
             picture = preparePicture(picture, file);
             pictureService.save(picture);
             model.addAttribute("uploadedImagePath", picture.getPath());
```

Comment:
to można wyjebać, chciałem, żeby po dodaniu wyświetlał się dodany obrazek ale w sumie to po chuj to